phlop/os.py

```
@@ -48,5 +48,21 @@ def write_to_file(file, contents, mode="w", skip_if_empty=True):
             raise RuntimeError(f"Failed to write to file {file}: {e}")
 
 
+def read_file(file):
+    try:
+        with open(file, "r") as f:
+            return f.read()
+    except IOError as e:
+        raise RuntimeError(f"Failed to read file {file}: {e}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)